static/service-worker-registration.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+if ('serviceWorker' in navigator) {
+  window.addEventListener('load', function() {
```

Comment:
@dtinth actually waiting for load event is better for installing service worker. see https://github.com/GoogleChrome/sw-precache/blob/master/demo/app/js/service-worker-registration.js